### PR TITLE
chore: remove lerna bootstrap

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

When install the stack a postinstall script launch lerna bootstrap.
This command link the stack dependencies.
So for example 

packages/containers/node_modules/react-talend-component -(is a link)-> packages/components

Because we use webpack this have the side effet to embed react multiple time in the build.
So we have in the storybook of containers the following issues:

> Uncaught Error: Invariant Violation: addComponentAsRefTo(...): Only a ReactOwner can have refs.

**What is the new behavior?**

Don't ever use again --learna bootstrap--
Because it create symlinks inside your node_modules.

So start by a cleanup before using this PRs.

```
lerna clean
```

Look at the added documentation in the README for more information.

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

File: README.md [hunk 1]
```diff
@@ -31,3 +31,55 @@ front-end stack.
...
+
+The stack as one may entry point:
+
+```
+yarn run watch
+```
+
+This watcher will trigger build and sync inside the stack for you.
+
+For example if you modify a component, it will rebuild component into lib folder and copy the content throw forms and containers.
```
Comment (jsomsanith) [inline README.md]: I don't even have to say it, I think you know what I want to say here :)

File: README.md [hunk 2]
```diff
@@ -31,3 +31,55 @@ front-end stack.
...
+
+```
+learna clean
+```
+
+or you will have bad behavior with the following tools.
+
+### yarn run watch
+
+The stack as one may entry point:
```
Comment (jsomsanith) [inline README.md]: `as one may entry point`--> `as one entry point`

File: README.md [hunk 3]
```diff
@@ -31,3 +31,55 @@ front-end stack.
 - [talend-icons](https://github.com/Talend/ui/tree/master/icons)
 - [bootstrap-talend-theme](https://github.com/Talend/ui/tree/master/theme)
 
+## Tools (dev environment)
+
+If you have done a learna bootstrap in the past, please start by
+
+```
+learna clean
```
Comment (jsomsanith) [inline README.md]: `learna`--> `lerna`

File: README.md [hunk 4]
```diff
@@ -31,3 +31,55 @@ front-end stack.
 - [talend-icons](https://github.com/Talend/ui/tree/master/icons)
 - [bootstrap-talend-theme](https://github.com/Talend/ui/tree/master/theme)
 
+## Tools (dev environment)
+
+If you have done a learna bootstrap in the past, please start by
```
Comment (jsomsanith) [inline README.md]: `learna`--> `lerna`

File: copylibs.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+		'*'
+	);
+
+	const dest = path.resolve(
+		'packages',
+		info.dest
+	);
+	console.log(`Copy ${source} -> ${dest}`);
+	const options = {
+	//	clean: true, // The flag to remove files that copied on past before copy
```
Comment (jsomsanith) [inline copylibs.js]: Commented code + empty `options` object

File: copylibs.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+	};
+	cpx[command](source, dest, options)
+	//.on('copy', (e) => process.stdout.write("."))
+	//.on('copy', (e) => console.log(`${e.srcPath} -> ${e.dstPath}`))
```
Comment (jsomsanith) [inline copylibs.js]: Commented code

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

**Comment by jmfrancois** [comment]
I have some bad behavior with the CI (seems a caching issue)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

File: .travis.yml [hunk 7]
```diff
@@ -21,11 +12,12 @@ node_js:
   - '6'
 before_install: .travis/before_install.sh
 before_script:
+  - echo yarn cache dir && yarn cache dir
```
Comment (talend-glorieux) [inline .travis.yml]: Please remove those debug lines.

File: .travis/before_install.sh [hunk 8]
```diff
@@ -6,6 +6,9 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
 	exit 0
 fi
 
+rm -rf /home/travis/.cache
+rm -rf /home/travis/.yarn-cache
```
Comment (talend-glorieux) [inline .travis/before_install.sh]: Why do you need to remove the cache?

File: copylibs.js [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+program
+	.version('0.0.1')
+	.option('-c, --clean', 'remove files that copied on past before')
+	.option('-d, --debug', 'display more info')
+	.option('-q, --quiet', 'display nothing')
+	.option('-s, --scope [scope]', 'only one deps')
+	.option('-w, --watch', 'copy and watch to copy again')
+	.parse(process.argv);
+
+const command = program.watch ? 'watch' : 'copy';
```
Comment (talend-glorieux) [inline copylibs.js]: Why is this defined so far away from where it's used?

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

File: README.md [hunk 10]
```diff
@@ -31,3 +31,49 @@ front-end stack.
...
+Just build all the packages for static purpose.
+It use the prepublish command.
+
+### copylibs
+
+```
+./copylibs.js
+```
+
+This script will copy all the *lib* folder of the stack into their dependencies.
```
Comment (frassinier) [inline README.md]: Folders

File: README.md [hunk 11]
```diff
@@ -31,3 +31,49 @@ front-end stack.
...
+
+This script for example will copy the *lib* folder of components into containers's node_modules and forms's node_modules.
```
Comment (frassinier) [inline README.md]: containers and forms node_modules*

File: README.md [hunk 12]
```diff
@@ -31,3 +31,49 @@ front-end stack.
...
+
+If the content of a lib folder change it triggers the copy.
+
+```
+./copylibs.js --scope=components
+```
+
+Only copy components into forms and containers.
+
+Same with all subfolder of packages.
```
Comment (frassinier) [inline README.md]: Subfolers

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://110.talend.surge.sh)

**Pull request merged**